ApplicationLibCode/ProjectDataModel/Summary/RimSummaryPlotSourceStepping.cpp

```diff
@@ -507,14 +496,12 @@ void RimSummaryPlotSourceStepping::fieldChangedByUi( const caf::PdmFieldHandle*
         {
             for ( auto curve : curves )
             {
-                if ( isYAxisStepping() )
                 {
```

Comment:
If the two `adr` variables are renamed to `adrX` and `adrY` you can drop the extra scoping.

Reply:
Fixed